<sdf version='1.6'> 
 	 <world name='default'> 
 	 	 <light name='sun' type='directional'> 
 	 	 	 <cast_shadows>1</cast_shadows> 
 	 	 	 <pose frame=''>0 0 10 0 -0 0</pose> 
 	 	 	 <diffuse>0.5 0.5 0.5 1</diffuse> 
 	 	 	 <specular>0.1 0.1 0.1 1</specular> 
 	 	 	 <attenuation> 
 	 	 	 	 <range>1000</range> 
 	 	 	 	 <constant>0.9</constant> 
 	 	 	 	 <linear>0.01</linear> 
 	 	 	 	 <quadratic>0.001</quadratic> 
 	 	 	 </attenuation> 
 	 	 	 <direction>0.9 0.5 -1</direction> 
 	 	 </light>
 
 	 	 <scene> 
 	 	 	 <background>130 200 255 1 </background>
 	 	 	 <ambient>130 140 140 1</ambient>
 	 	 </scene>
 
 	 	 <include>
 	 	 	 <uri>model://Sign</uri>
 	 	 	 <pose>12 1 0 0 0 0</pose>
 	 	 	 <static>1</static>
 	 	 </include>
 
 	 	 <include>
 	 	 	 <uri>model://Sign</uri>
 	 	 	 <pose>-4 1 0 0 0 0</pose>
 	 	 	 <static>1</static>
 	 	 </include>
 
	 	 <include> 
 	 	 	 <uri>model://dirt_plane</uri> 
 	 	 </include>
 
	 	 <population name="sticks"> 
 	 	 	 <model name="Red_stick">
 	 	 	 	 <include>
 	 	 	 	 	 <static>0</static>
 	 	 	 	 	 <uri>model://Red_stick</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	<pose>4.500000 -0.350000 0.1 0 0 0</pose>
 	 	 	<distribution> 
 	 	 	 	 <type>grid</type> 
 	 	 	 	 <rows>3</rows>
 	 	 	 	 <cols>2</cols>
 	 	 	 	 <step>7.000000 0.700000 0</step>
 	 	 	 </distribution>
 	 	 </population>
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>1.500000 0.340049 0.15 0 0 -89.556070</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>1.750000 0.337765 0.15 0 0 -110.116102</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>2.000000 0.363369 0.15 0 0 120.320336</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>2.250000 0.343087 0.15 0 0 -62.213648</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>2.500000 0.360783 0.15 0 0 97.043774</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>2.750000 0.361950 0.15 0 0 107.550331</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>3.000000 0.353523 0.15 0 0 31.708378</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>3.250000 0.335640 0.15 0 0 -129.235692</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>3.500000 0.345598 0.15 0 0 -39.616462</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>3.750000 0.351024 0.15 0 0 9.213065</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>4.000000 0.347226 0.15 0 0 -24.962572</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>4.250000 0.361811 0.15 0 0 106.298874</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>4.500000 0.367789 0.15 0 0 160.104943</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>4.750000 0.346668 0.15 0 0 -29.984400</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>5.000000 0.364711 0.15 0 0 132.397805</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>5.250000 0.352867 0.15 0 0 25.799911</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>5.500000 0.349469 0.15 0 0 -4.779896</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>5.750000 0.359018 0.15 0 0 81.161296</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>6.000000 0.369497 0.15 0 0 175.473302</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>6.250000 0.330420 0.15 0 0 -176.219698</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>6.500000 0.350259 0.15 0 0 2.330678</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>6.750000 0.350946 0.15 0 0 8.515498</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>7.000000 0.332887 0.15 0 0 -154.013629</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>7.250000 0.337479 0.15 0 0 -112.686628</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>1.450321 -0.358479 0.15 0 0 -76.309871</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>1.700321 -0.341054 0.15 0 0 80.510344</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>1.950321 -0.364030 0.15 0 0 -126.272006</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>2.200321 -0.366961 0.15 0 0 -152.646987</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>2.450321 -0.349612 0.15 0 0 3.489539</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>2.700321 -0.353077 0.15 0 0 -27.689399</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>2.950321 -0.359813 0.15 0 0 -88.312650</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>3.200321 -0.355656 0.15 0 0 -50.908417</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>3.450321 -0.355963 0.15 0 0 -53.665899</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>3.700321 -0.339359 0.15 0 0 95.768373</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>3.950321 -0.367530 0.15 0 0 -157.774271</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>4.200321 -0.330875 0.15 0 0 172.125107</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>4.450321 -0.358854 0.15 0 0 -79.688195</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>4.700321 -0.362607 0.15 0 0 -113.461494</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>4.950321 -0.332424 0.15 0 0 158.183934</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>5.200321 -0.366377 0.15 0 0 -147.393074</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>5.450321 -0.360419 0.15 0 0 -93.772573</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>5.700321 -0.344502 0.15 0 0 49.479673</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>5.950321 -0.341327 0.15 0 0 78.056427</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>6.200321 -0.355282 0.15 0 0 -47.534293</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>6.450321 -0.360856 0.15 0 0 -97.705430</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>6.700321 -0.343054 0.15 0 0 62.516791</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>6.950321 -0.364280 0.15 0 0 -128.521294</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>7.200321 -0.364771 0.15 0 0 -132.935793</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>1.408750 -1.057223 0.15 0 0 -65.009566</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>1.658750 -1.059333 0.15 0 0 -83.998640</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>1.908750 -1.054672 0.15 0 0 -42.046972</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>2.158750 -1.053950 0.15 0 0 -35.550192</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>2.408750 -1.036325 0.15 0 0 123.077755</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>2.658750 -1.051246 0.15 0 0 -11.212937</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>2.908750 -1.030915 0.15 0 0 171.763369</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>3.158750 -1.067783 0.15 0 0 -160.044793</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>3.408750 -1.049451 0.15 0 0 4.937080</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>3.658750 -1.047793 0.15 0 0 19.865756</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>3.908750 -1.052470 0.15 0 0 -22.232971</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>4.158750 -1.064880 0.15 0 0 -133.920598</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>4.408750 -1.047032 0.15 0 0 26.709202</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>4.658750 -1.053592 0.15 0 0 -32.331919</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>4.908750 -1.064187 0.15 0 0 -127.683087</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>5.158750 -1.063063 0.15 0 0 -117.571001</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>5.408750 -1.049121 0.15 0 0 7.914247</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>5.658750 -1.057811 0.15 0 0 -70.295319</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>5.908750 -1.030774 0.15 0 0 173.033678</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>6.158750 -1.061298 0.15 0 0 -101.679653</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>6.408750 -1.063616 0.15 0 0 -122.544216</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>6.658750 -1.052447 0.15 0 0 -22.024834</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>6.908750 -1.033662 0.15 0 0 147.042474</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<include> 
 	 	 	 <uri>model://Leek</uri> 
 	 	 	 <pose>7.158750 -1.046258 0.15 0 0 33.677370</pose> 
 	 	 	 <static>0</static> 
 	 	 </include> 
 
	 	<population name="Rocks1">
 	 	 	 <model name="Rock">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Rock</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>4.500000 0.000000 -0.01 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>8 0.600000 0.01</size>
 	 	 	 </box>
 	 	 	 <model_count>30</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population> 
 
	 	<population name="Big_rocks1">
 	 	 	 <model name="Big_rock">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Big_rock</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>4.500000 0.200000 -0.01 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>6 0.2 0.001</size>
 	 	 	 </box>
 	 	 	 <model_count>12</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population> 
 
	 	<population name="Rocks2">
 	 	 	 <model name="Rock">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Rock</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>4.500000 -0.700000 -0.01 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>8 0.600000 0.01</size>
 	 	 	 </box>
 	 	 	 <model_count>30</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population> 
 
	 	<population name="Big_rocks2">
 	 	 	 <model name="Big_rock">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Big_rock</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>4.500000 -0.500000 -0.01 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>6 0.2 0.001</size>
 	 	 	 </box>
 	 	 	 <model_count>12</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population> 
 
	 	<population name="Grass">
 	 	 	 <model name="Grass">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Grass</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>4.500000 -0.350000 -0.01 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>8 1.400000 0.05</size>
 	 	 	 </box>
 	 	 	 <model_count>36</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population> 
 
	 	<population name="Grass1">
 	 	 	 <model name="Grass">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Grass</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>4.500000 0.350000 -0.01 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>8 0.100000 0.05</size>
 	 	 	 </box>
 	 	 	 <model_count>12</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population> 
 
	 	<population name="Grass2">
 	 	 	 <model name="Grass">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Grass</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>4.500000 -0.350000 -0.01 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>8 0.100000 0.05</size>
 	 	 	 </box>
 	 	 	 <model_count>12</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population> 
 
	 	<population name="Grass3">
 	 	 	 <model name="Grass">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Grass</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>4.500000 -1.050000 -0.01 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>8 0.100000 0.05</size>
 	 	 	 </box>
 	 	 	 <model_count>12</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population> 
 
	 	 <population name="Trees1"> 
 	 	 	 <model name="Tree">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Tree</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>17 0 -0.6 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>5 20 3</size>
 	 	 	 </box>
 	 	 	 <model_count>5</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population>
 	 	 
 	 	 <population name="Trees2">
 	 	 	 <model name="Tree">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Tree</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>0 -17 -0.6 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>30 6 3</size>
 	 	 	 </box>
 	 	 	 <model_count>5</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population>
 
 	 	 <population name="Trees3">
 	 	 	 <model name="Tree">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Tree</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>0 15 -0.6 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>30 6 3</size>
 	 	 	 </box>
 	 	 	 <model_count>7</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 	 </population>
 
 	 	 <population name="Trees4">
 	 	 	 <model name="Tree">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Tree</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>-15 0 -0.6 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>10 30 3</size>
 	 	 	 </box>
 	 	 	 <model_count>8</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population>
 
	 </world> 
 </sdf>